Added blog on Sieve of Eratosthenes

aastha01chauhan opened the pull request:
I have added a blog on sieve of eratosthenes, which also has a pseudo java code. Please let me know if you expect any changes and if not merge this PR and add hacktoberfest label on it.

Ruchip16 commented on _posts/SieveOfEratosthenes.md:
```diff
@@ -0,0 +1,132 @@
+
+
+## Introduction
+t is easy to find if some number (say N) is prime or not — you simply need to check if at least one number from numbers lower or equal sqrt(n) is divisor of N. This can be achieved by simple code:
```
It is easy*

Ruchip16 commented on _posts/SieveOfEratosthenes.md:
```diff
@@ -0,0 +1,132 @@
+
+But there is great idea — why to "do not check even numbers". This idea can be extended to: "do not divide N by candidate numbers, but mark the prime multiples as 'not prime'".
+
+In other words: if we know about some number p, taht it is prime, we mark all multiples (2*p, 3*p, 4*p, ...) as not a prime. To implement this we need limit (max number we care about), let say it is 120 (because there is nice animation of algorithm on wikipedia).
```
In other words: if we know about some number p, **that** it is prime, we mark all multiples (2*p, 3*p, 4*p, ...) as not a prime. To implement this we need a limit (max number we care about), let say it is 120 (because there is a nice animation of the algorithm on Wikipedia). - plz run your blog through Grammarly once it has a lot of grammatical mistakes

Ruchip16 replied:
(because there is a nice animation of the algorithm on Wikipedia) - this line is not needed

Ruchip16 commented on _posts/SieveOfEratosthenes.md:
```diff
@@ -0,0 +1,132 @@
+
+In other words: if we know about some number p, taht it is prime, we mark all multiples (2*p, 3*p, 4*p, ...) as not a prime. To implement this we need limit (max number we care about), let say it is 120 (because there is nice animation of algorithm on wikipedia).
+
+In Java I'm using this code:
```
highlight your code using the code-blocks like this 👇 
```
add your code here
```

Ruchip16 commented on _posts/SieveOfEratosthenes.md:
```diff
@@ -0,0 +1,132 @@
+
+
+## Conclusion
+The running time of block sieving is the same as for regular sieve of Eratosthenes (unless the size of the blocks is very small), but the needed memory will shorten to O(n‾√+S) and we have better caching results. On the other hand, there will be a division for each pair of a block and prime number from [1;n‾√], and that will be far worse for smaller block sizes. Hence, it is necessary to keep bal ...
```
Run your blog through Grammarly once so that the grammatical spelling mistakes are avoided and add your code in code-blocks otherwise it won't look good when the blog will get published

LoopThrough-i-j commented:
Also add yourself, as an author @aastha01chauhan.

aastha01chauhan commented:
@LoopThrough-i-j added my name is authors' file. Please add hackberfest-accepted label too!

Ruchip16 commented:
> @LoopThrough-i-j added my name is authors' file. Please add hackberfest-accepted label too!

plz do the requested changes first & commit them @aastha01chauhan

aastha01chauhan commented:
@Ruchip16 I added the codeblock, checked for grammatical mistakes, and added my name in author, have committed the changes too. Is there anything you expect me change?

aastha01chauhan commented:
Checked for mistakes again and corrected too

LoopThrough-i-j commented:
@aastha01chauhan, please add yourself to the authors as well, as mentioned in the docs.

After completing changes run:
```
git add .
git commit -m "Fix: blog on Sieve of Eratosthenes"
git push
```
